samples/bluetooth/peripheral_ams_client/prj.conf

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_HEAP_MEM_POOL_SIZE=1024
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+CONFIG_BT=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_DEVICE_NAME="AMS"
```

Comment:
To be in line with our other samples this name should be `Nordic_AMS` unless the site specification requires that name exactly

Reply:
When the device name is set to Nordic_AMS, the advertising data is unable to include the complete name. The name field is truncated to "Nordic_A". Any comment?

Reply:
You could put uuid adv data in a scan response packet if it is possible in your sample then you would use a longer device name.

Reply:
Will do.